scripts/zones/Attohwa_Chasm/mobs/Alastor_Antlion.lua

```diff
@@ -0,0 +1,69 @@
+-----------------------------------
+-- Area: Attohwa Chasm
+-- NPC:  Alastor Antlion
+-- ID: 	
+-----------------------------------
+-----------------------------------
+
+require("scripts/globals/titles");
+require("scripts/globals/status");
+require("scripts/globals/magic");
+
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+	mob:setMobMod(MOBMOD_ADD_EFFECT,mob:getShortID());
+	mob:setTP(100);

```

Comment:
setTP() won't work in onMobInitialize, you'll have to put it in onMobSpawn if you want it spawned with TP.